platform/viewer/src/customHooks/useServer.js

```diff
@@ -32,11 +32,22 @@ const getServers = (appConfig, project, location, dataset, dicomStore) => {
       wadoRoot: pathUrl,
     };
     servers = GoogleCloudUtilServers.getServers(data, dicomStore);
+    if (!isValidServer(servers[0], appConfig)) {
+      return;
+    }
   }
 
   return servers;
 };
 
+const isValidServer = (server, appConfig) => {
+  if (appConfig.enableGoogleCloudAdapter) {
+    return GoogleCloudUtilServers.isValidServer(server);
+  } else {
```

Comment:
I would have espected eslint to flag this: https://eslint.org/docs/rules/no-else-return

Reply:
ok